ReactWindows/ReactNative.Shared/Modules/Storage/AsyncStorageConstants.cs

```diff
@@ -0,0 +1,135 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using Newtonsoft.Json.Linq;
+
+namespace ReactNative.Modules.Storage
+{
+    class AsyncStorageConstants
```

Comment:
>AsyncStorageConstants [](start = 10, length = 21)

Constants is not really a good name for this anymore... maybe just consolidate it with AsyncStorageErrorHelpers and call it AsyncStorageHelpers?